Linked List/LinkedList.playground/Contents.swift

```diff
@@ -317,6 +317,60 @@ extension LinkedList: ExpressibleByArrayLiteral {
   }
 }
 
+// MARK: - Collection
+extension LinkedList : Collection {
+  
+  public typealias Index = LinkedListIndex<T>
+  
+  /// The position of the first element in a nonempty collection.
+  ///
+  /// If the collection is empty, `startIndex` is equal to `endIndex`.
+  /// - Complexity: O(1)
+  public var startIndex: Index {
+    get {
```

Comment:
The getter is implicit for all computed properties. No need to explicitly write `get`.